make/protogen.mk

```diff
@@ -118,7 +120,7 @@ GRPC_GATEWAY_DIR = $(shell go list -f '{{.Dir}}' -m github.com/grpc-ecosystem/gr
 .PHONY: proto-fmt
 proto-fmt: $(PROTOC_GEN_LINT)
 	@echo "Checking for proto style errors"
-	@PATH=$(PROTO_GOBIN) $(PROTOC) \
+	$(SILENT)PATH=$(PROTO_GOBIN) $(PROTOC) \
```

Comment:
why `$(SILENT)` here but `@` above?

Reply:
same for some others below this

Reply:
Because the `echo` is never interesting from a debug perspective